# ----------------------------------
# 跟踪模块给底层执行的通信消息

Header  header
uint64  vehicle_id
float64 target_steering # 轮偏角，弧度
float64 target_speed    # 秒速

# ----------------------------------